portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -1281,57 +1274,81 @@ definition {
 			virtualHost = "www.baker.com",
 			webId = "www.baker.com");
 
+		MyAccount.openMyAccountAdmin(baseURL = "http://www.test.com:8080");
+
+		User.editUserInformation(userScreenNameEdit = "localhost");
+
+		User.logoutPG();
+
 		User.firstLoginUI(
 			password = "test",
-			specificURL = "http://www.able.com:9080",
+			specificURL = "http://www.able.com:8080",
 			userEmailAddress = "test@www.able.com");
 
-		User.openUsersAdmin(baseURL = "http://www.able.com:9080");
+		User.openUsersAdmin(baseURL = "http://www.able.com:8080");
 
 		User.editEmailAddressCP(
 			password = "test",
 			userEmailAddress = "test@liferay.com",
 			userScreenName = "test");
 
-		User.logoutPG(specificURL = "http://www.able.com:9080");
+		User.logoutPG(specificURL = "http://www.able.com:8080");
 
 		User.viewLoggedOutPG();
 
 		User.firstLoginUI(
 			password = "test",
-			specificURL = "http://www.baker.com:9080",
+			specificURL = "http://www.baker.com:8080",
 			userEmailAddress = "test@www.baker.com");
 
-		User.openUsersAdmin(baseURL = "http://www.baker.com:9080");
+		User.openUsersAdmin(baseURL = "http://www.baker.com:8080");
 
 		User.editEmailAddressCP(
 			password = "test",
 			userEmailAddress = "test@liferay.com",
 			userScreenName = "test");
 
-		User.logoutPG(specificURL = "http://www.baker.com:9080");
+		User.logoutPG(specificURL = "http://www.baker.com:8080");
 
 		User.viewLoggedOutPG();
 
 		User.firstLoginUI();
 
 		SAML.configureLiferaySAMLAsIdP(samlEntityID = "samlidp");
 
+		User.logoutPG();
+
 		SAML.configureLiferaySAMLAsSP(
 			samlEntityID = "samlsp",
-			spURL = "http://www.able.com:9080");
+			spAttributes = '''emailAddress
+				firstName
+				lastName
+				screenName
+				uuid''',
+			spURL = "http://www.able.com:8080",
+			userEmailAddress = "test@www.able.com",
```

Comment:
@weller0246, According to the test's original design, at this point, what will be the Instance admin's email address?